include/key_table.h

```diff
@@ -25,6 +25,11 @@ typedef struct Keys
 
 extern KeyTable key_table;
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+  
+#ifndef APP
```

Comment:
I notice here you've used APP and in other places you use SGX. Is there a distinction between the two?  

In both cases, these are generic terms that might conflict with marcos used by dependencies. We had that issue at one point with Kmyth and a tss2 dependency. I suggest using something like PELZ_SGX to reduce the chance of collision.

Reply:
I've changed those names to `PELZ_SGX` and `PELZ_APP`. They are different, `PELZ_SGX` ensures code that is only appropriate to the enclave is only included in the enclave, and that code that isn't appropriate in the enclave isn't built into the enclave. `PELZ_APP` is similar, but applies to the untrusted portion.

Reply:
I ended up renaming these (again) to PELZ_SGX_TRUSTED and PELZ_SGX_UNTRUSTED to make their use clearer.